test/cljs/status_im/test/utils/ethereum/abi_spec.cljs

```diff
@@ -2,6 +2,16 @@
   (:require [cljs.test :refer-macros [deftest is testing]]
             [status-im.utils.ethereum.abi-spec :as abi-spec]))
 
+(deftest hex-to-number
+  (testing "hex number is less than 53 bits, it returns a number")
+  (is (= (abi-spec/hex-to-number "9")
+         9))
+  (is (= (abi-spec/hex-to-number "99999999")
+         2576980377))
```

Comment:
Why not always a string?

Reply:
convenience, in a lot of cases an integers below 53 bits is expected